app/models/pawn.rb

```diff
@@ -17,12 +17,16 @@ def valid_move?(destination_row, destination_column)
     elsif move == FORWARD_TWO && first_move?
       return !occupied_space?(destination_row, destination_column)
     elsif CAPTURE_MOVES.include?(move)
-      return capturable?(destination_row, destination_column)
+      return can_capture?(destination_row, destination_column)
     end
 
     false
   end
 
+  def previously_moved_two_spaces?
+    (row - previous_row).abs == 2
+  end
```

Comment:
@mscoutermarsh I actually originally had this as a private method but the tests were failing because it was complaining about not being able to access a private method when it was being called on the opponent piece in the en_passant? method

```
def en_passant?(destination_column)
    opponent_pawn = game.pieces.find_by(type: 'Pawn', color: opposite_color, row: row, column: destination_column)
    opponent_pawn.present? && opponent_pawn.previously_moved_two_spaces? && opponent_pawn.last_to_move
end
```
The `opponent_pawn.previously_moved_two_spaces?` was giving the problem, I imagine since the opponent piece is a separate instance so it can't access that private method from the current instance(is that right?), so I just decided to move the previously_moved_two_spaces? out of the private section.